src/index.ts

```diff
@@ -1,8 +0,0 @@
-/*
```

Comment:
I'm fairly confident that we need to keep this file for two reasons: 
1. it's used as the `exports` in the package.json.
2. This file existing allows `@oclif/core` to more performantly find plugin roots at run time